src/esp/gfx/GenericDrawable.cpp

```diff
@@ -118,11 +116,8 @@ void GenericDrawable::updateShaderLightingParameters(
 
 void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
                            Mn::SceneGraph::Camera3D& camera) {
-  if (!isRendererCreated) {
```

Comment:
It turns out that this block is not necessary because the `GenericDrawable::draw` codepath is impossible to reach when `createRenderer==false`.